examples/helper-methods-part-1.ipynb

```diff
@@ -0,0 +1,10676 @@
+       "            return {value:v,enumerable:false,writable:true,configurable:true};\n",
+       "        };\n",
+       "        global.Base64.extendString = function () {\n",
+       "            Object.defineProperty(\n",
+       "                String.prototype, 'fromBase64', noEnum(function () {\n",
+       "                    return decode(this)\n",
+       "                }));\n",
+       "            Object.defineProperty(\n",
+       "                String.prototype, 'toBase64', noEnum(function (urisafe) {\n",
+       "                    return encode(this, urisafe)\n",
+       "                }));\n",
+       "            Object.defineProperty(\n",
+       "                String.prototype, 'toBase64URI', noEnum(function () {\n",
+       "                    return encode(this, true)\n",
+       "                }));\n",
+       "        };\n",
+       "    }\n",
+       "    //\n",
+       "    // export Base64 to the namespace\n",
+       "    //\n",
+       "    if (global['Meteor']) { // Meteor.js\n",
+       "        Base64 = global.Base64;\n",
+       "    }\n",
+       "    // module.exports and AMD are mutually exclusive.\n",
+       "    // module.exports has precedence.\n",
+       "    if (typeof module !== 'undefined' && module.exports) {\n",
+       "        module.exports.Base64 = global.Base64;\n",
+       "    }\n",
+       "    else if (typeof define === 'function' && define.amd) {\n",
+       "        // AMD. Register as an anonymous module.\n",
+       "        define([], function(){ return global.Base64 });\n",
+       "    }\n",
+       "    // that's it!\n",
+       "    return {Base64: global.Base64}\n",
+       "}));\n",
+       "\n",
+       "function format(value) {\n",
+       "  if (Array.isArray(value)) {\n",
+       "    const [first, second] = value;\n",
+       "    if (first === -Infinity) {\n",
+       "      return `< ${formatValue(second)}`;\n",
+       "    }\n",
+       "    if (second === Infinity) {\n",
+       "      return `> ${formatValue(first)}`;\n",
+       "    }\n",
+       "    return `${formatValue(first)} - ${formatValue(second)}`\n",
+       "  }\n",
+       "  return formatValue(value);\n",
+       "}\n",
+       "\n",
+       "function formatValue(value) {\n",
+       "  if (typeof value === 'number') {\n",
+       "    return formatNumber(value);\n",
+       "  }\n",
+       "  return value;\n",
+       "}\n",
+       "\n",
+       "function formatNumber(value) {\n",
+       "  const digits = 2;\n",
+       "  if (Math.abs(value) > 10000) {\n",
+       "    return value.toExponential(digits);\n",
+       "  } else if (!Number.isInteger(value)) {\n",
+       "    return value.toLocaleString(undefined, {\n",
+       "      minimumFractionDigits: digits,\n",
+       "      maximumFractionDigits: digits\n",
+       "    });\n",
+       "  } else {\n",
+       "    return value.toLocaleString();\n",
+       "  }\n",
+       "}\n",
+       "\n",
+       "const responsive = doc
```

Comment:
How about passing `palette='[red, blue]'` (or whatever colors are usually used for GOP and DEM) instead of the default here?

Reply:
@andy-esch as mentioned, that (and more!) come in part 2!